skiko/src/commonMain/kotlin/org/jetbrains/skia/PathSegmentIterator.kt

```diff
@@ -56,11 +58,57 @@ class PathSegmentIterator internal constructor(val _path: Path?, ptr: NativePoin
     }
 }
 
+private fun NativePointer.nextSegment() = pathSegmentFromFloatArray(withResult(FloatArray(10)) {
+    PathSegmentIterator_nNext(this, it)
+})
+
+
+private fun pathSegmentFromFloatArray(points: FloatArray): PathSegment {
+    val context = points.last().toInt()
+
+    val verb = (context) and ((1 shl 3) - 1)
```

Comment:
Why not convert to enum here?

Reply:
Yep, let's check again enum value rather than its ordinal counterpart - pushed the fix